### PR TITLE
Section timeframe pageviews

### PR DESCRIPTION
**Pull request by katsuraku**

### CODE REVIEW

**Comment by radiodario** [comment]
link to trello card?

File: src/server/aggregations/SectionMetadata.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ export default function SectionMetadataAggregation(query) {
               min_doc_count: 0
         }
       },
-      topics_covered: {
-        cardinality: {
-          field: "topics_not_analyzed"
+      "topics_covered": {
+        "cardinality": {
+          "field": "topics_not_analyzed"
```
Comment (radiodario) [inline src/server/aggregations/SectionMetadata.js]: we don't need the quotes as this is a javascript object, so we leave them out for consistency

File: src/server/aggregations/SectionMetadata.js [hunk 2]
```diff
@@ -20,9 +20,9 @@ export default function SectionMetadataAggregation(query) {
           size : 10
         }
       },
-      distinct_articles: {
-        cardinality: {
-          field: "article_uuid"
+      "distinct_articles_published": {
+        "cardinality": {
+          "field": "article_uuid"
```
Comment (radiodario) [inline src/server/aggregations/SectionMetadata.js]: same thing here - quotes aren't needed.

You can remove them easily with the following regexp:
`s/"(.*)"\s?:/\1:/`

File: src/server/aggregations/SectionMetadataComparator.js [hunk 3]
```diff
@@ -1,19 +1,14 @@
...
         field: "topics_not_analyzed",
         size : 10
       }
     },
-    distinct_articles: {
-      cardinality: {
-        field: "article_uuid"
+    "distinct_articles_published": {
+      "cardinality": {
+        "field": "article_uuid"
```
Comment (radiodario) [inline src/server/aggregations/SectionMetadataComparator.js]: any reasoning behind the name change for these fields?

File: src/server/utils/universalDataFormatter.js [hunk 4]
```diff
@@ -76,6 +76,7 @@ const fields = {
   sectionViews: {name: 'aggregations.section_views', formatter: format},
   topicsCovered: 'aggregations.topics_covered.value',
   sectionsCovered: 'aggregations.sections_covered.value',
+  articlePublishCount: 'aggregations.distinct_articles_published.value',
   articleCount: 'aggregations.distinct_articles.value',
```
Comment (radiodario) [inline src/server/utils/universalDataFormatter.js]: should you delete this one as it's been renamed?

**Comment by radiodario** [comment]
The section comparator data looks wrong, especially when compared against the same section... perhaps not to do with your PR but it's noticeable?

aside from that, it looks good.

File: src/server/aggregations/SectionMetadataComparator.js (hunk 3 above)
Comment (katsuraku) [inline src/server/aggregations/SectionMetadataComparator.js]: it's a new, separate aggregation. Yes quotation marks will come out :+1:

File: src/server/utils/universalDataFormatter.js (hunk 4 above)
Comment (katsuraku) [inline src/server/utils/universalDataFormatter.js]: The first one is still being used in other places in the code

**Comment by katsuraku** [comment]
Section comparator data is indeed wrong at the moment. Another card is being worked on to correct it. We did a check yesterday to make sure the branches would work together and they looked good.

**Pull request merged**